system/Debug/Exceptions.php

```diff
@@ -98,6 +98,9 @@ public function __construct(ExceptionsConfig $config, $request, ResponseInterfac
      * handling of our application.
      *
      * @codeCoverageIgnore
+     *
+     * @return void
+     * @phpstan-return never|void
```

Comment:
It seems `never` is not returned. See https://phpstan.org/writing-php-code/phpdoc-types#bottom-type
```suggestion
```

Reply:
I changed there to `@phpstan-return void` .

Reply:
If `@return` and `@phpstan-return` are the same, `@phpstan-return` is not needed.
Remove it.

Reply:
done.